Horatio/Operations/AlertOperation.swift

```diff
@@ -64,24 +64,39 @@ class AlertOperation: Operation {
     }
 
     override func execute() {
-        var presentationViewController: UIViewController?
-        
-        if let presentationContext = presentationContext {
-            presentationViewController = presentationContext
-        } else {
-            presentationViewController = UIApplication.shared.keyWindow?.rootViewController
+        // it's probably not safe to even walk the view hierarchy from a background thread, so do this all on main
+        DispatchQueue.main.async {
```

Comment:
Hmm, I'm always wary of sync, due to the risk of deadlocks. Do you think it provides an advantage in this case?